lerobot/templates/visualize_dataset_template.html

```diff
@@ -2,217 +2,355 @@
-      video.addEventListener('timeupdate', () => (timer.textContent = formatTime(video.currentTime) + ' / ' +
-        formatTime(video.duration)));
-      const formatTime = (time) => {
-        var hours = Math.floor(time / 3600);
-        var minutes = Math.floor((time % 3600) / 60);
-        var seconds = Math.floor(time % 60);
-        return (hours > 0 ? hours + ':' : '') + (minutes < 10 ? '0' + minutes : minutes) + ':' + (seconds < 10 ?
-          '0' + seconds : seconds);
-      }
-
-      btnPlay.addEventListener('click', () => {
-        videos.forEach(video => video.play());
-        btnPlay.classList.toggle('hidden');
-        btnPause.classList.toggle('hidden');
-      });
-
-      btnPause.addEventListener('click', () => {
-        videos.forEach(video => video.pause());
-        btnPlay.classList.toggle('hidden');
-        btnPause.classList.toggle('hidden');
-      });
-
-      btnRewind.addEventListener('click', () => {
-        videos.forEach(video => (video.currentTime = 0.0));
-      });
-
-      btnBackward.addEventListener('click', () => {
-        videos.forEach(video => (video.currentTime -= 5));
-      });
-
-      btnForward.addEventListener('click', () => {
-        videos.forEach(video => (video.currentTime += 5));
-      });
-
-      videos.forEach(video => {
-        video.addEventListener('ended', () => {
-          btnPlay.classList.remove('hidden');
-          btnPause.classList.add('hidden');
-        });
-      });
-
-      // Update the video time when the slider value changes
-      slider.addEventListener('input', function () {
-        const sliderValue = slider.value;
-        videos.forEach(video => {
-          const time = (video.duration * sliderValue) / 100;
-          video.currentTime = time;
-        });
-      });
-
-      // Synchronize slider with the video's current time
-      const syncSlider = (video) => {
-        video.addEventListener('timeupdate', function () {
-          if (video.duration) {
-            const pc = (100 / video.duration) * video.currentTime;
-            slider.value = pc;
-            dygraphTime = video.currentTime;
-            const index = Math.floor(pc * dygraph.numRows() / 100);
-            dygraph.setSelection(index, undefined, true, true);
-          }
-        });
-      };
-      videos.forEach(video => syncSlider(video));
-
-      // hit "Space" to play/pause
-      window.addEventListener('keydown', e => {
-        if (e.keyCode === 32 || e.key === ' ') {
-          e.preventDefault();
-          btnPause.classList.contains("hidden") ? btnPlay.click() : btnPause.click();
+
+    <script>
+        class SeededRandom {
+            static seed = 42;  // Initial seed
+
+            static random() {
+                const a = 1664525;
+                const c = 1013904223;
+                const m = 2 ** 32;
+                SeededRandom.seed = (a * SeededRandom.seed + c) % m;
+                return SeededRandom.seed / m;
+            }
```

Comment:
Unused?

Reply:
indeed. deleted in 28ea5e7c0236b1350a005a88332af0ba132ae14d